src/include/ipc4/fw_reg.h

```diff
@@ -119,7 +119,7 @@ union ipc4_rom_info {
 	} platform;
 } __attribute__((packed, aligned(4)));
 
-/* Number of pipeline registers slots in FW Regs. */
+/* Number of dsp core supported in FW Regs. */
```

Comment:
while at it - what is "FW Regs?"

Reply:
the FW status exposed to host via shared memory. This could be missing though and could be added as a comment at the top of this header file.